src/Microsoft.DotNet.Build.Tasks/Targets/test-runtime/project.json

```diff
@@ -43,6 +43,6 @@
        "dnxcore50": { }
     },
     "runtimes": {
-       "win7-x86": { }
+       "win7-x64": { }
```

Comment:
cc @ericstj 
You would do that so that someone could pass in the other runtime identifier. My understanding here is that all this does is tell the restore command what set of items to prepopulate in the lock file so that they can be resolved by our build system. It is saying I want to know about both of those runtimes.